practice-app/src/todos/Metrics.tsx

```diff
@@ -29,7 +29,7 @@ export default function Metrics({ todos }: { todos: ToDo[] }) {
         <h2 className="text-sm text-slate-100">Done</h2>
         <p className="text-3xl text-slate-100">
           {/* TODO: display total count of todos that are completed */}
```

Comment:
@akhand08 the code looks good to me! Could you please remove the `TODO` comment as the task here is done?